sopel/modules/currency.py

```diff
@@ -1,102 +1,189 @@
 # coding=utf-8
 # Copyright 2013 Elsie Powell, embolalia.com
+# Copyright 2019 Mikkel Jeppesen
 # Licensed under the Eiffel Forum License 2
 from __future__ import unicode_literals, absolute_import, print_function, division
 
+import time
 import re
+import requests
 
-from requests import get
-from sopel.module import commands, example, NOLIMIT
+from sopel.module import commands, example, NOLIMIT, rule
+from sopel.tools import stderr
+from sopel.config.types import StaticSection, ValidatedAttribute
 
-# The Canadian central bank has better exchange rate data than the Fed, the
-# Bank of England, or the European Central Bank. Who knew?
-base_url = 'https://www.bankofcanada.ca/valet/observations/FX{}CAD/json'
+rates_fiat_json = {}
+rates_btc_json = {}
+
+
+fiat_url = 'https://api.exchangeratesapi.io/latest?base=EUR'
```

Comment:
How do you feel about capitalized constants? Should the URLs be changed? e.g., `fiat_url` to `FIAT_URL`?

Reply:
It's a nice style guideline to follow, yeah. :+1:

I think in addition to being capitalized, `regex` should become something a bit more specific, like `EXCHANGE_REGEX` or similar. (Yes, I know that name is a holdover from the original code. It was kinda bad then, too, lol. I didn't touch the variables in my cruft cleanup commit [1cfe818] because, well, I forgot—but I can *totally* pretend that it's because I wanted to minimize merge conflicts with this PR! :stuck_out_tongue:)

Reply:
Done :)